src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs

```diff
@@ -6,23 +6,63 @@
+    /// ## Examples
+    /// The following code example demonstrates how to use the <xref:System.Diagnostics.ProcessStartInfo> class to start Internet Explorer, providing the destination URLs as <xref:System.Diagnostics.ProcessStartInfo> arguments.
+    /// [!code-cpp[Process.Start_static#1](~/samples/snippets/cpp/VS_Snippets_CLR/Process.Start_static/CPP/processstartstatic.cpp)]
+    /// [!code-csharp[Process.Start_static#1](~/samples/snippets/csharp/VS_Snippets_CLR/Process.Start_static/CS/processstartstatic.cs)]
+    /// [!code-vb[Process.Start_static#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/Process.Start_static/VB/processstartstatic.vb)]
+    /// ]]></format></remarks>
+    /// <altmember cref="System.Diagnostics.Process"/>
     public sealed partial class ProcessStartInfo
     {
         private string? _domain;
 
+        /// <summary>Gets or sets the user password in clear text to use when starting the process.</summary>
+        /// <value>The user password in clear text.</value>
         [SupportedOSPlatform("windows")]
         public string? PasswordInClearText { get; set; }
 
+        /// <summary>Gets or sets a value that identifies the domain to use when starting the process. If this value is <see langword="null" />, the <see cref="System.Diagnostics.ProcessStartInfo.UserName" /> property must be specified in UPN format.</summary>
+        /// <value>The Active Directory domain to use when starting the process. If this value is <see langword="null" />, the <see cref="System.Diagnostics.ProcessStartInfo.UserName" /> property must be specified in UPN format.</value>
+        /// <remarks>This property is primarily of interest to users within enterprise environments that use Active Directory.</remarks>
         [SupportedOSPlatform("windows")]
         public string Domain
         {
             get => _domain ?? string.Empty;
             set => _domain = value;
         }
 
+        /// <summary>Gets or sets a value that indicates whether the Windows user profile is to be loaded from the registry.</summary>
+        /// <value><see langword="true" /> if the Windows user profile should be loaded; otherwise, <see langword="false" />. The default is <see langword="false" />.</value>
+        /// <remarks>This property is referenced if the process is being started by using the user name, password, and domain.
+        /// If the value is <see langword="true" />, the user's profile in the `HKEY_USERS` registry key is loaded. Loading the profile can be time-consuming. Therefore, it is best to use this value only if you must access the information in the `HKEY_CURRENT_USER` registry key.
+        /// In Windows Server 2003 and Windows 2000, the profile is unloaded after the new process has been terminated, regardless of whether the process has created child processes.
+        /// In Windows XP, the profile is unloaded after the new process and all child processes it has created have been terminated.</remarks>
```

Comment:
There is no particular need to clean up the docs while we do this, of course. But it is funny to see mention of things as old as Windows XP in here. I guess in general, we expect there to be comments like this in our code that do not apply to .NET Core at all?

(And what about comments on API that we simply don't have any more for some reason, that's got to be very rare of course.)

Reply:
> we expect there to be comments like this in our code that do not apply to .NET Core at all?

Yes, I'm porting everything unmodified. But also keep in mind that MS Docs had one single docs section for APIs that exist in both .NET Core and .NET Framework. This particular API you pointed out exists in both versions, so it makes sense to preserve this comment.

For APIs that _only_ exist in .NET Framework, we decided their source of truth will be MS Docs, since we won't be able to backport it to source.